### ISSUE TITLE
[Bug]: Exception when going to navigation properties

### ISSUE THREAD

**Issue by GAL-GC** [pimcore/pimcore#12151]
### Expected behavior

When creating or editing a document, I want to set navigation settings, for example the document language or exclude it from navigation.

### Actual behavior

It happened, that an exception occured, when I went to Navigation & properties tab.

```
Status: 500 | Internal Server Error
URL: /admin/page/save?task=autoSave
Method: PUT
Message: Pimcore\Bundle\AdminBundle\Controller\Admin\Document\DocumentControllerBase::getPropertyInheritance(): Argument #3 ($propertyValue) must be of type string, null given, called in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/DocumentControllerBase.php on line 138
Trace: 
in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/DocumentControllerBase.php:306
#0 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/DocumentControllerBase.php(138): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\DocumentControllerBase->getPropertyInheritance(Object(Pimcore\Model\Document\Page), 'language', NULL)
#1 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/PageController.php(446): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\DocumentControllerBase->addPropertiesToDocument(Object(Symfony\Component\HttpFoundation\Request), Object(Pimcore\Model\Document\Page))
#2 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/DocumentControllerBase.php(445): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\PageController->setValuesToDocument(Object(Symfony\Component\HttpFoundation\Request), Object(Pimcore\Model\Document\Page))
#3 /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/Document/PageController.php(155): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\DocumentControllerBase->saveDocument(Object(Pimcore\Model\Document\Page), Object(Symfony\Component\HttpFoundation\Request))
#4 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(152): Pimcore\Bundle\AdminBundle\Controller\Admin\Document\PageController->saveAction(Object(Symfony\Component\HttpFoundation\Request), Object(Pimcore\Document\StaticPageGenerator))
#5 /var/www/html/vendor/symfony/http-kernel/HttpKernel.php(74): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#6 /var/www/html/vendor/symfony/http-kernel/Kernel.php(202): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#7 /var/www/html/public/index.php(36): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#8 {main}
```

### Steps to reproduce

Unfortunately it doesn't seem to happen every time, but it occured already a few times, when I created documents and switched to the navigation tab, as I was about to define there a language.

Edit:

I was able to reproduce it on the root document. It has no language set. When I add a new property on navigation settings, I get this error.

**Comment by passioneight** [comment]
I can confirm this behaviour. For me, it was independent of any properties. After a language was set for the document, the error disappeared.

### Steps to reproduce
1. Open the `Home` (i.e., Document with ID 1)
2. Try saving.

**Comment by blankse** [comment]
I created a PR: https://github.com/pimcore/pimcore/issues/12195

**Comment by aryaantony92** [comment]
Fixed by #12195
